src/Catalog/Helpers/Utils.cs

```diff
@@ -35,6 +35,19 @@ public static class Utils
 
         private static readonly ConcurrentDictionary<string, string> _resourceStringCache = new ConcurrentDictionary<string, string>();
 
+        private static readonly char[] TagTrimChars = { ',', ' ', '\t', '|', ';' };
+
+        public static string[] SplitTags(string original)
```

Comment:
This is so feed2catalog and db2azuresearch use the same tag splitting and normalizing logic.